code/Classes/Permissions/ArekHealth.swift

```diff
@@ -30,9 +30,9 @@ open class ArekHealth: ArekBasePermission, ArekPermissionProtocol {
     
     public var identifier: String = "ArekHealth"
     
-    var hkObjectType: HKObjectType?
-    var hkSampleTypesToShare: Set<HKSampleType>?
-    var hkSampleTypesToRead: Set<HKSampleType>?
+    public var hkObjectType: HKObjectType?
+    public var hkSampleTypesToShare: Set<HKSampleType>?
+    public var hkObjectTypesToRead: Set<HKObjectType>?
```

Comment:
Hi @wordlessj ,
why do we use `HKSampleType` to share and `HKObjectType` to read?

Reply:
It's the signature of the request function on `HKHealthStore`.

```
func requestAuthorization(toShare typesToShare: Set<HKSampleType>?, read typesToRead: Set<HKObjectType>?, completion: @escaping (Bool, Error?) -> Void)
```